packages/react-components/src/Concepts/Positioning/PositioningBestPractices.md

```diff
@@ -0,0 +1,6 @@
+# Best practices
+
+⚠️ Simpler configuration is better. Positioning is complicated, do not make your life harder by using it too much ⚠️
```

Comment:
What is purpose of this statement? 😺 In current state I would rather remove it completely.